pytest_testrail/plugin.py

```diff
@@ -27,11 +27,13 @@
 
 ADD_RESULTS_URL = 'add_results_for_cases/{}'
 ADD_TESTRUN_URL = 'add_run/{}'
+ADD_PLANENTRY_URL = '/add_plan_entry/{}'
```

Comment:
Please, could you remove the leading `/` to be homogeneous with the rest of the code ?

Reply:
Corrected, along with adding `_URL` to suites URL